Solutions/BitSight/Analytic Rules/BitSightGetWorkFromData.yaml

```diff
@@ -0,0 +1,35 @@
+id: 54e77434-2184-435f-bc87-52b832ee451f
```

Comment:
Please remove this query as discussed before.

Reply:
@devikamehra @v-atulyadav 
After internal discussion we have decided to remove the Work From Home use case completely.
And as a action we have removed the Analytics Rule, Playbook and Workbook related to Work From Home.

Thanks.